$NetBSD: patch-xx,v 1.8 2008/10/11 09:56:38 uebayasi Exp $

Following breaks gtk option as prefers /usr/X11R6 libs to those in /usr/pkg,
and is not needed as these get set in enough other places - at least for
pkgsrc.

--- src/s/netbsd.h.orig	2008-10-10 10:35:49.000000000 +0900
+++ src/s/netbsd.h
@@ -102,12 +102,6 @@ Boston, MA 02110-1301, USA.  */
 #define AMPERSAND_FULL_NAME
 
 #ifdef __ELF__
-/* Here is how to find X Windows.  LD_SWITCH_X_SITE_AUX gives an -R option
-   says where to find X windows at run time.  We convert it to a -rpath option
-   which is what OSF1 uses.  */
-#define LD_SWITCH_SYSTEM_tmp `echo LD_SWITCH_X_SITE_AUX | sed -e 's/-R/-Wl,-rpath,/'`
-#define LD_SWITCH_SYSTEM LD_SWITCH_SYSTEM_tmp -Wl,-rpath,/usr/pkg/lib -L/usr/pkg/lib -Wl,-rpath,/usr/local/lib -L/usr/local/lib
-
 /* The following is needed to make `configure' find Xpm, Xaw3d and
    image include and library files if using /usr/bin/gcc.  That
    compiler seems to be modified to not find headers in
